spec/models/vmdb_database/seeding_spec.rb

```diff
@@ -1,99 +1,157 @@
 describe VmdbDatabase do
-  context "::Seeding" do
-    include_examples(".seed called multiple times")
+  describe "::Seeding" do
     let(:connection) { ApplicationRecord.connection }
 
-    it ".seed" do
-      MiqDatabase.seed
-      vmdb_database = double('vmdb_database')
-      allow(described_class).to receive(:seed_self).and_return(vmdb_database)
-      expect(vmdb_database).to receive(:seed)
-      described_class.seed
+    describe ".seed" do
+      let!(:db) { described_class.seed }
+
+      it "creates tables, text_tables, and indexes" do
```

Comment:
This test is intentionally long winded instead of breaking it up into multiple tests per concern because of how long a full seed takes.  All of the vmdb_* tests take 7 seconds on my machine, but if I split this out, you're looking at another 2 seconds per test.